core/unit_test/TestInit.hpp

```diff
@@ -70,4 +70,7 @@ TEST(TEST_CATEGORY, dispatch) { test_dispatch<TEST_EXECSPACE>(); }
 }  // namespace Test
 
 #include <TestCompilerMacros.hpp>
+#ifndef KOKKOS_ENABLE_OPENACC
+// FIXME_OPENACC: not implemented yet.
```

Comment:
```suggestion
#ifndef KOKKOS_ENABLE_OPENACC. // FIXME_OPENACC not implemented
```
put on a single line
same comment everywhere

Reply:
Actually in this case say "TeamPolicy not implement"